src/OSeq.mli

```diff
@@ -200,6 +200,10 @@ val find : ('a -> bool) -> 'a t -> 'a option
 (** [find p e] returns the first element of [e] to satisfy [p],
     or None. *)
 
+val find_map : ('a -> 'b option) -> 'a t -> 'b option
+(** [find_map f e] returns the result of [f] on the first element of [e]
+    for which it returns [Some _], or [None] otherwise. *)
```

Comment:
can you add a `@since NEXT_RELEASE` flag? :)

Reply:
done